styles.css

```diff
@@ -1,136 +1,527 @@
+    top: 50%;
+    transform: translateY(-50%);
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+  }
+
+  .logo__circle_color_dark-blue {
+    background: #0056EC;
+  }
+
+  .logo__circle_color_blue {
+    left: 8px;
+    top: 56%;
+    background: #C8DBFB;
+    opacity: .7;
+  }
+
+  .menu {
+    display: flex;
+    align-items: center;
+  }
+
+  .menu__item {
+    position: relative;
+    margin-right: 26px;
+    font: 500 14px/17px Roboto, arial, helvetica, sans-serif;
+    color: #000;
+    text-decoration: none;
+    transition: opacity .5s;
+  }
+
+  .menu__item:after {
+    position: absolute;
+    left: 0;
+    display: block;
+    content: '';
+    width: 0;
+    height: 2px;
+    background: #0056EC;
+    opacity: 0;
+    transition: opacity .5s, width .5s;
+  }
+
+  .menu__item:hover {
+    opacity: .5;
+  }
+
+  .menu__item:hover:after, .menu__item_active:after {
+    opacity: 1;
+    width: 100%;
+  }
+
+  .menu__item_active::after {
+    position: absolute;
+    left: 0;
+    display: block;
+    content: '';
+    width: 100%;
+    height: 2px;
+    background: #0056EC;
+  }
+
+  .menu__item:hover:after {
+    position: absolute;
+    left: 0;
+    display: block;
+    content: '';
+    width: 100%;
+    height: 2px;
+    background: #0056EC;
+  }
+
+  .menu__item:last-child {
+    margin-right: 0;
+  }
+
+  .main {
+    padding: 80px 58% 65px 0;
+    background: bottom 10px right no-repeat url(img/mainImg-1170.svg);
+  }
+
+  .main__title {
+    margin: 0 0 36px 0;
+    font: 50px/66px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__title-text_highlight {
+    background: #ECF3FF;
+  }
+
+  .main__subtitle {
+    margin: 0 0 43px 0;
+    font: 18px/21px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    display: inline-block;
+    padding: 15px 21px;
+    font: 14px/17px Roboto, arial, helvetica, sans-serif;
+    text-decoration: none;
+    color: #fff;
+    background: #0056EC;
+    border-radius: 4px;
+    transition: opasity .5s;
+  }
+
+  .main__button:hover {
+    opacity: .5;
+  }
 }
+
+@media (max-width: 375px) {
+  body {
+    margin: 0;
+  }
+
+  .wrapper {
+    margin: auto;
+    width: 87%;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 8px;
+  }
+
+  .logo {
+    position: relative;
+    padding: 24px 0;
+    padding-left: 44px;
+    font: 24px/31px pt_serifbold, arial, helvetica, sans-serif;
+    color: white;
+  }
+
+  .logo__circle {
+    position: absolute;
+    left: 0;
+    top: 50%;
+    transform: translateY(-50%);
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+  }
+
+  .logo__circle_color_dark-blue {
+    background: #0056EC;
+  }
+
+  .logo__circle_color_blue {
+    left: 8px;
+    top: 56%;
+    background: #C8DBFB;
+    opacity: .7;
+  }
+
+  .span {
+    position: relative;
+    top: 40px;
+    width: 25px;
+    height: 3px;
+    background-color: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Board/pullRequest?prid=6207323)